site/pages/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Head from "next/head";
+import {
+    faChevronDown,
+    faChevronUp,
+    faUpRightFromSquare,
+} from "@fortawesome/free-solid-svg-icons";
+import { Menu, Transition } from "@headlessui/react";
+import { useState, useRef, useEffect } from "react";
+export default function Page() {
+    const [chevron, setChevron] = useState(false);
+    const menuButtonRef = useRef<HTMLButtonElement | null>(null);
+    const toggleDropdown = () => {
+        setChevron(!chevron);
+    };
+    const handleClickOutside = (event: MouseEvent) => {
+        if (menuButtonRef.current && !menuButtonRef.current.contains(event.target as Node)) {
+            setChevron(false);
+        }
+    };
+    useEffect(() => {
+        document.addEventListener('click', handleClickOutside);
+
+        return () => {
+            document.removeEventListener('click', handleClickOutside);
+        };
+    }, []);
+    return (
+        <>
+            <Head>
+                <title>Burrow</title>
+                <meta
+                    name="description"
+                    content="Burrow is an open-source tool for bypassing firewalls, built by teenagers at HackClub."
+                />
+                <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+            </Head>
+            <div className="flex min-h-screen items-center justify-center">
+                <div className="mb-48 text-center">
+                    <h1 className="font-PhantomSans text-5xl md:text-6xl lg:text-7xl xl:text-8xl 2xl:text-9xl">
+                        <span className="font-bold text-hackClubRed">Burrow</span> Through{" "}
+                        Firewalls
+                    </h1>
+                    <div className="mx-auto my-2 w-11/12 font-PhantomSans text-lg md:my-10 md:w-3/4 lg:text-xl xl:text-2xl 2xl:text-3xl">
+                        <p>
+                            Burrow is an open source tool for burrowing through firewalls,
+                            built by teenagers at{" "}
+                            <span className="text-hackClubRed underline">
+                                <a href="https://www.hackclub.com/" target="_blank">
+                                    Hack Club.
+                                </a>
+                            </span>{" "}
+                            <span className="rounded-md bg-burrowHover p-0.5 font-SpaceMono text-hackClubBlue">
+                                burrow
+                            </span>{" "}
+                            provides a simple command-line tool to open virtual interfaces and
+                            direct traffic through them.
```

Comment:
I don't think the second sentence adds to the first. Potential second sentence could be 

```suggestion
                            Burrow is an open source tool for burrowing through firewalls,
                            built by teenagers at{" "}
                            <span className="text-hackClubRed underline">
                                <a href="https://www.hackclub.com/" target="_blank">
                                    Hack Club.
                                </a>
                            </span>{" "}
                            <span className="rounded-md bg-burrowHover p-0.5 font-SpaceMono text-hackClubBlue">burrow</span>{" "} is a Rust-based VPN for getting around restrictive Internet censors.
```

I'm not attached to it though, so please feel free to suggest alternatives!

Reply:
Works! I just copied it from GitHub